src/main/java/marquez/api/GetNamespaceResponse.java

```diff
@@ -0,0 +1,53 @@
+package marquez.api;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+
+public class GetNamespaceResponse {
+
+  final String name;
+  final String ownerName;
+  final String description;
+
+  @JsonCreator
+  public GetNamespaceResponse(
+      final String name,
```

Comment:
This works without `@JsonProperty("name")`?

Reply:
Good point. I'll add it.

Reply:
We'll get more clarity on this once we have integration tests that work with data.